app/views/prof_landing/index.html.erb

```diff
@@ -0,0 +1,105 @@
+<!-- app/views/prof_landing/landing.html.erb -->
+<html>
+  <head>
+    <title>Professor Landing Page</title>
+    <style>
+      /* Light Mode Styles */
+      body,
+      html {
+        margin: 0;
+        padding: 0;
+        overflow: hidden;
+      }
+      body {
+        background-color: #fff;
+        color: #333;
+      }
+      .form-control {
+        background-color: #f8f9fa; /* Light gray for input fields */
+        color: #333;
+      }
+      .btn.btn-danger.custom-maroon-button {
+        background-color: maroon;
+        border-color: maroon;
+        color: #fff;
+      }
+      a,
+      label {
+        color: #007bff; /* Blue for links and labels in light mode */
+      }
+
+      /* Dark Mode Styles */
+      html.dark-mode body {
+        background-color: #333;
+        color: #fff;
+      }
+      html.dark-mode .form-control {
+        background-color: #444;
+        color: #fff;
+      }
+      html.dark-mode .btn.btn-danger.custom-maroon-button {
+        background-color: maroon;
+        border-color: maroon;
+        color: #fff;
+      }
+      html.dark-mode a,
+      html.dark-mode label {
+        color: #bbb; /* Lighter color for links and labels in dark mode */
+      }
+
+      .vertical-center-container {
+        display: flex;
+        flex-direction: column;
+        align-items: center; /* Center horizontally */
+        justify-content: center; /* Center vertically */
+        height: 100vh; /* Make the container full height of the viewport */
+        overflow: hidden; /* Hide overflow content */
+      }
+
+      .vertical-center-container a {
+        margin-right: 0;
+        margin-bottom: 10px;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="d-flex justify-content-between align-items-center">
+      <button
+        href="#"
+        class="btn btn-danger custom-maroon-button ml-3"
+        id="back-button"
+      >
+        Log Out
+      </button>
+      <!-- <h1>Welcome, Professor!</h1> -->
+      <button class="btn btn-danger custom-maroon-button" id="dark-mode-toggle">
+        Toggle Dark Mode
+      </button>
+    </div>
+    <div class="buttons">
+      <div class="vertical-center-container">
+        <h1>Welcome, Professor!</h1>
+        <a href="#" class="btn btn-primary mb-2">View Results</a>
```

Comment:
<img width="1470" alt="Screenshot 2023-09-28 at 4 16 52 PM" src="https://github.com/Capstone-Matching/capstone-matching/assets/142866638/21fd67eb-741b-4293-84d1-550010fd69a2">

Can we have equal sized buttons and match the color theme. Let's not have multiple colours.

Reply:
agreed!, use the maroon buttons we have in the application CSS for this

Reply:
I was thinking that the 'remove project' button should be red, as it signifies danger. The 'add project' button should be green, as it represents something positive. If we keep all the buttons the same color, it would be too monotonous.

Reply:
Updated the code for the UI to look like this
![image](https://github.com/Capstone-Matching/capstone-matching/assets/43250823/2e42719d-76a2-4248-909c-bae1d5145dc8)

Reply:
looks good, on the actual pages we can color code buttons (ie Remove and Add) to be certain colors, but I think this makes the professor home page UI look better